### PR TITLE
Restructure The Team folder

### LINKED ISSUE #2220: People shouldn't exist
Not sure why https://osu.ppy.sh/help/wiki/People/ exists when it's just The_team minus some.

Just redirect to The_team and call it a day?

Also consider moving inactive below user groups

### PR DESCRIPTION
**Pull request by TPGPL**
Resolves #2220 

---

### CODE REVIEW

**Comment by MegaApplePi** [comment]
Did we want to move Community Contributors under The Team? I thought that Community Contributors weren't necessarily part of The Team.

Also, there is no link to the Community Contributors from The Team page.

File: wiki/The_Team/en.md [hunk 1]
```diff
@@ -60,15 +60,19 @@ The people listed below are the core of the osu!team, and are chiefly responsibl
...
 | Name | Description |
 | :-- | :-- |
-| [Beatmap Nominators](/wiki/People/Beatmap_Nominators) | Users going above and beyond the call to ensure your maps get qualified |
-| [Developers](/wiki/People/Developers) | Making the game awesome by adding new features and fixing the bugs |
-| [Global Moderation Team](/wiki/People/Global_Moderation_Team) | Keeping watch over the forums and in-game chat |
-| [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) | Managing the Beatmap Nominators |
-| [osu! Alumni](/wiki/People/osu!_Alumni) | Those known for their contributions who have since moved on |
-| [Project Loved Team](/wiki/People/Project_Loved_Team) | Recognising the beatmaps that the community loves most |
-| [Support Team](/wiki/People/Support_Team) | Help / Feature Request assistance |
+| [Beatmap Nominators](/wiki/The_Team/Beatmap_Nominators) | Users going above and beyond the call to ensure your maps get qualified |
```
Comment (cl8n) [inline wiki/The_Team/en.md]: this is missing links from `People` now, some of which (Community Contributors?) I'm not really sure belong here anyway as @MegaApplePi mentioned. this is one of the reasons I didn't support the change in the issue's thread

File: wiki/The_Team/es.md [hunk 2]
```diff
@@ -52,14 +52,17 @@ Las personas que nombradas a continuación son el núcleo del equipo de osu!, y
...
-| [Beatmap Nomination Group](/wiki/People/Beatmap_Nomination_Group) | Encargados de la calificación de beatmaps |
+| [Global Moderation Team](/wiki/The_Team/Global_Moderation_Team) | Vigilando los foros y el chat en el juego |
+| [Beatmap Nomination Group](/wiki/The_Team/Beatmap_Nomination_Group) | Encargados de la calificación de beatmaps |
 | [Developers](https://osu.ppy.sh/groups/11) | Haciendo que el juego sea impresionante al agregar nuevas funciones y corregir los errores |
-| [osu! Alumni](/wiki/People/osu!_Alumni) | Aquellos conocidos por sus contribuciones que ya no están activos |
-| [Support Team Redux](/wiki/People/Support_Team) | Asistencia en foros de ayuda |
-| [Quality Assurance Team](/wiki/People/Quality_Assurance_Team) | Te ayudamos a disfrutar de osu!, empezando por asegurar la calidad de los beatmaps. |
+| [osu! Alumni](/wiki/The_Team/osu!_Alumni) | Aquellos conocidos por sus contribuciones que ya no están activos |
+| [Support Team Redux](/wiki/The_Team/Support_Team) | Asistencia en foros de ayuda |
+| [Quality Assurance Team](/wiki/The_Team/Quality_Assurance_Team) | Te ayudamos a disfrutar de osu!, empezando por asegurar la calidad de los beatmaps. |
```
Comment (cl8n) [inline wiki/The_Team/es.md]: same deal as my comment about "Beatmap_Nomination_Group", this redirect doesn't (and shouldn't) exist so you need to fix the link

**Comment by cl8n** [comment]
I don't think this is a good idea because of what @MegaApplePi said and what I've said at https://github.com/ppy/osu-wiki/issues/2220#issuecomment-488714517

if you want all of the osu! team stuff grouped together, I guess you could do
```
People
├── The_Team
│   ├── Beatmap_Nominators
│   └── ... (other groups that are part of the team)
├── Community_Contributors
└── ... (other people/groups *not* part of the team)
```
but this is kind of messy because (for example) the other pages under beatmap nominators are going to be nested really far down. I prefer the current version

also there are PRs almost ready to merge rn that translate parts of The_Team and People, jsyk you will have to solve conflicts in a few days probably

**Comment by TPGPL** [comment]
closing as its not a good solution rn

**Pull request closed without merging**